examples/example.md

```diff
@@ -141,160 +165,163 @@ provider-ionos-cf2fec81b474-54f5d7ddd4-w9w9h        1/1     Running   0
 provider-kubernetes-df601dea646a-84f7d6db54-t5dn5   1/1     Running   0          65m
 ```
 
-Check CRDs:
+#### Check CRDs
+
+To check the CRDs, run the following command:
 
 ```bash
 kubectl get crds | grep ionoscloud
 ```
 
-A CRD named `postgresclusters.dbaas.ionoscloud.crossplane.io` should be displayed in the output.
+A CRD named ``postgresclusters.dbaas.ionoscloud.crossplane.io`` should be displayed in the output.
 
-Next, we will create a Custom Resource(CR) of type `postgresclusters.dbaas.ionoscloud.crossplane.io` in order to
+After that, you can create a Custom Resource (CR) of type ``postgresclusters.dbaas.ionoscloud.crossplane.io`` to
 provision a DBaaS Postgres Cluster in the IONOS Cloud.
 
-## Provision DBaaS Postgres Cluster
+### Provision DBaaS Postgres cluster
 
-For the DBaaS Postgres Service, there is only Cluster resource available into the Crossplane Provider IONOS Cloud.
+For the DBaaS Postgres service, there is only cluster resource available into the Crossplane Provider IONOS Cloud.
 
-❗ Before running the next command, make sure to **update** the values in
-the `examples/ionoscloud/dbaas/postgres-cluster.yaml` file. Look for `spec.forProvider` fields. It is required to
-specify the Datacenter (via ID or via reference), Lan (via ID or via reference), CIDR, and location(in sync with the
-Datacenter) and also credentials for the database user.
+{% hint style="warning" %}
+**Warning:** Before running the next command, make sure to **update** the values in
+the ``examples/ionoscloud/dbaas/postgres-cluster.yaml`` file. Look for ``spec.forProvider`` fields. It is required to specify the datacenter (via ID or via reference),LAN (via ID or via reference), CIDR, and location(in sync with the datacenter) and also credentials for the database user.
+{% endhint %}
 
-### Create Datacenter, Lan, Postgres Cluster CRs
+### Create datacenter CR, LAN, and Postgres Cluster CRs
 
-Create a Datacenter CR, a Lan CR and a Postgres Cluster CR - using the next command
+To create a datacenter CR, LAN CR and Postgres Cluster CR, run the following command:
 
 ```bash
 kubectl apply -f examples/ionoscloud/dbaas/postgres-cluster.yaml
 ```
 
-### Get Datacenter, Lan, Postgres Cluster CRs
+### Get Datacenter, LAN, and Postgres Cluster CRs
```

Comment:
Should `datacenter` be lowercase here like above? Or Uppercase everywhere?

Reply:
@DiPoojary I need your suggestion here :)

Reply:
Since it's a resource type, I suggest using Uppercase, like for LAN and Postgres Cluster.

Reply:
Okay sounds good :) Let me update it.

Reply:
Hi @Barshad96 - Please change it to **Get Data Center**

Here is an [example](https://docs.ionos.com/cloud/getting-started/basic-tutorials/data-center-basics)